mast/templates/personal_dashboard.html

```diff
@@ -1,21 +1,23 @@
 {% extends "main_layout_template.html" %}
```

Comment:
Please use modification from my most recent branch in order to avoid conflicts.

Reply:
Not sure what modification are you referring to since I did not modify this file it should be automatically  updated by your branch commit.